src/app/fdctl/run/tiles/fd_sender.c

```diff
@@ -48,10 +49,9 @@ struct fd_voter_tile_ctx {
   fd_shred_dest_weighted_t * new_dest_ptr;
   ulong                      new_dest_cnt;
 
-  uchar vote_update_buf[ FD_TXN_MTU ];
-  ulong vote_update_sz;
+  uchar txn_buf[ sizeof(fd_txn_p_t) ] __attribute__((aligned(alignof(fd_txn_p_t))));
 
-  fd_gossip_peer_addr_t tpu_vote_serve_addr;
+  fd_gossip_peer_addr_t tpu_serve_addr;
```

Comment:
is this right? should it have both tpu and tpu vote?

Reply:
it just needs any port number to set as the source port. In the configuration of this tile as the voter, it is our tpu_vote port.